src/main/webapp/app/shared/util/firebase/firebase-path-utils.ts

```diff
@@ -36,3 +36,12 @@ export const extractArrayPath = (valuePath: string) => {
   const firebaseArrayPath = pathParts.join('/');
   return { firebaseArrayPath, deleteIndex };
 };
+
+export const getImplicationPathFromRCTPath = (rctPath: string) => {
+  if (rctPath.endsWith('excludedRCTs')) {
+    const rctPathParts = rctPath.split('/');
+    rctPathParts.pop();
+    return rctPathParts.join('/');
+  }
+  return null;
```

Comment:
Can we add some comments here why we do this?

Reply:
The logic was part of initial refactoring effort, but unused now. I removed this method. Thanks!